docs/index.md

```diff
@@ -88,10 +88,9 @@ The DC's 'sdc' zone (there should be only one) handles taking hourly dumps
 of most of the SDC API's models and uploading those to Manta. Here is how
 that works (see TOOLS-278 for background):
 
-- Every minute and hourly cron job (at the top of the hour) dumps the output of
+- Every hourly cron job (at the top of the hour) dumps the output of
   the various APIs to "/var/log/sdc-data/*.json":
 
-        * * * * * /opt/smartdc/sdc/tools/dump-minutely-sdc-data.sh >>/var/log/dump-minutely-sdc-data.log 2>&1
```

Comment:
Should delete this "tools/dump-minutely-sdc-data.sh" file as well.

Reply:
Drop thjis file as well: https://github.com/joyent/sdc-sdc/blob/TRITON-2132/probes/sdc/sdc/minutely-dump-error.json

Reply:
done